### DISCUSSION TITLE
Which open-zwave version am I running?

### DISCUSSION THREAD

**Discussion by embcla** [hassio-addons/addon-zwave2mqtt#85] [Q&A] [answered]
# Problem/Motivation

I don't understand how to keep track of the underlying open-zwave implementation.
Can somebody explain it?

## Expected behavior

A quick and easy way to track the integrated zwave2mqtt version as well as openzwave version.

## Actual behavior

I can't find the info I'm looking for

## Steps to reproduce

> (How can someone else make/see it happen)

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by frenck** [comment, marked as answer]
OpenZwave:

https://github.com/hassio-addons/addon-zwave2mqtt/blob/6044a6fd4a4e2b8bc25b5a7aee23d829a0ebabf4/zwave2mqtt/Dockerfile#L26

Zwave2MQTT:

https://github.com/hassio-addons/addon-zwave2mqtt/blob/6044a6fd4a4e2b8bc25b5a7aee23d829a0ebabf4/zwave2mqtt/Dockerfile#L40

Of course, changes are listed in the release notes:

<https://github.com/hassio-addons/addon-zwave2mqtt/releases>
